### PR TITLE
refactor: re-write clipboard module test to kotlin

### PR DESCRIPTION
**Pull request by hurali97**
## Summary:

Migrates the **ClipboardModuleTest** Java class to Kotlin. It also changes the import for **ClipboardManager** from `android.text.ClipboardManager` to `android.content.ClipboardManager`. Since the import from `android.text` is deprecated and in the [ClipboardManager.java](https://github.com/facebook/react-native/blob/main/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/clipboard/ClipboardModule.java) the import refers to `android.content`, so it's safe to update the import in test file as well.

It's a sub-task of the umbrella issue [here](https://github.com/facebook/react-native/issues/37708)

## Changelog:

[INTERNAL] [CHANGED] ClipboardModuleTest class migrated from Java to Kotlin

## Test Plan:

🟢 Test passes when running the following:

```sh
./gradlew :packages:react-native:ReactAndroid:test
```

### CODE REVIEW

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/clipboard/ClipboardModuleTest.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+
+@Suppress("DEPRECATION")
+@SuppressLint("ClipboardManager", "DeprecatedClass")
+@RunWith(RobolectricTestRunner::class)
+class ClipboardModuleTest {
+  private lateinit var mClipboardModule: ClipboardModule
+  private lateinit var mClipboardManager: ClipboardManager
```
Comment (cortinico) [inline packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/clipboard/ClipboardModuleTest.kt:27]:
```suggestion
  private lateinit var clipboardModule: ClipboardModule
  private lateinit var clipboardManager: ClipboardManager
```

**Comment by hurali97** [comment]
> Is missing in this PR? Did you decided to don't do it? May I ask why?

Hey I think there is some misunderstanding. What I meant was, I updated the import to reference `android.content.ClipboardManager`, so now we don't have the deprecated `import android.text.ClipboardManager` anymore. The PR was updated already !!


Also, addressed the changes for naming convention. Ready for re-review 👍

**Comment by cortinico** [comment]
> > Is missing in this PR? Did you decided to don't do it? May I ask why?
> 
> Hey I think there is some misunderstanding. What I meant was, I updated the import to reference `android.content.ClipboardManager`, so now we don't have the deprecated `import android.text.ClipboardManager` anymore. The PR was updated already !!
> 
> Also, addressed the changes for naming convention. Ready for re-review 👍

Ah ok I understand. Sorry I misread your initial comment

**Review by cortinico — APPROVED** [review]
LGTM 👍

**Pull request closed without merging**